config/crd/bases/apps.3scale.net_apicasts.yaml

```diff
@@ -275,9 +275,34 @@ spec:
                 description: OpenSSLPeerVerificationEnabled controls OpenSSL peer
                   verification.
                 type: boolean
+              openTelemetry:
+                description: OpenTelemetry contains the gateway instrumentation configuration
+                  with APIcast.
+                properties:
+                  enabled:
+                    description: Enabled controls whether OpenTelemetry integration
+                      with APIcast is enabled. By default it is not enabled.
+                    type: boolean
+                  tracingConfigSecretKey:
+                    description: TracingConfigSecretKey if unspecified, the first
+                      key in lexicographical order will be referenced as tracing configuration.
+                      If specified, the referenced value will be used as tracing configuration.
+                    type: string
+                  tracingConfigSecretRef:
+                    description: TracingConfigSecretRef contains a Secret reference
+                      the Opentelemetry configuration. The configuration file specification
+                      is defined in the Nginx instrumentation library repo https://github.com/open-telemetry/opentelemetry-cpp-contrib/tree/main/instrumentation/nginx
+                    properties:
+                      name:
+                        description: 'Name of the referent. More info: https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#names
+                          TODO: Add other useful fields. apiVersion, kind, uid?'
```

Comment:
future work or forgotten?

Reply:
that text is inherited from the `v1.LocalObjectReference`being part of `k8s.io/api@v0.24.3/core/v1/types.go`